private/buf/cmd/buf/buf_test.go

```diff
@@ -1158,12 +1158,12 @@ func TestConfigInitBasic(t *testing.T) {
 	testConfigInit(
 		t,
 		`version: v1
-lint:
-  use:
-    - DEFAULT
 breaking:
-  use:
-    - FILE
+    use:
+        - FILE
+lint:
+    use:
+        - DEFAULT
```

Comment:
Gah yea this isn't great, especially because it effects how users interact with `buf config init`. Unless there's a `yaml.Marshal` option we can take advantage of to control indentation, this might require that the `buf config init` path takes on a different path than the configuration written to the module cache.

Unfortunately, the aesthetic of the configuration we write with `buf config init` matters here.

Check out https://pkg.go.dev/gopkg.in/yaml.v3#Encoder.SetIndent and see if we can get that to work.

Reply:
Nice -- this works, so I'm happy about this :D